### ISSUE TITLE
User generic with TypeVarTuple does not check for minimal type of arguments

### ISSUE THREAD

**Issue by serhiy-storchaka** [python/cpython#99382]
The following code:
```python
from typing import *
T = TypeVar('T')
T2 = TypeVar('T2')
Ts = TypeVarTuple('Ts')
class A(Generic[T, T2, *Ts]):
    x: List[T]
    y: List[T2]
    z: Tuple[*Ts]

A[int]
```
is executed without errors. It is expected to get an error because `A` requires at least two arguments.

Without `TypeVarTuple` you get helpful errors.
```python
class B(Generic[T, T2]):
    pass

B[int]
B[int, str, bytes]
```
```
TypeError: Too few arguments for <class '__main__.B'>; actual 1, expected 2
```
```
TypeError: Too many arguments for <class '__main__.B'>; actual 3, expected 2
```

**Comment by sobolevn** [comment]
The code says this:

https://github.com/python/cpython/blob/f531b6879b530515b009ac79767702829848cf07/Lib/typing.py#L1833-L1836

**Comment by AlexWaygood** [comment]
I believe that this has been fixed and backported (in #99412 and #99866). Closing as completed.

(Feel free to reopen if I'm incorrect.)
